### DISCUSSION TITLE
Add Docker image of mmdeploy for Jetson Devices.  为mmdeploy增加Jetson平台Docker镜像

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#673] [MMDeploy]
# Add Docker image of mmdeploy for Jetson Devices.  
 为mmdeploy增加Jetson平台Docker镜像 

To ease the deployment with mmdeploy on Jetson devices, you need to write a Dockerfile, build and test the image and then create a PR to mmdeploy.
You can refer to the following resources:
1. [mmdeploy base Dockerfile](https://github.com/open-mmlab/mmdeploy/blob/main/docker/Base/Dockerfile)
2. [mmdeploy prebuild Dockerfile](https://github.com/open-mmlab/mmdeploy/blob/main/docker/Release/Dockerfile)
3. [nvidia l4t-pytorchDocker Image](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/l4t-pytorch)

为了方便用户在Jetson平台轻松部署模型，请编写适配Jetson平台的Dockerfile，构建和测试mmdeploy镜像
参考资料
1. [mmdeploy 基础Dockerfile](https://github.com/open-mmlab/mmdeploy/blob/main/docker/Base/Dockerfile)
2. [mmdeploy 预编译Dockerfile](https://github.com/open-mmlab/mmdeploy/blob/main/docker/Release/Dockerfile)
3. [nvidia l4t-pytorch镜像](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/l4t-pytorch)

## Task Information

**Repository**：MMDeploy

**Difficulty**：Hard

**Points**：60

**Estimated Time Consumption**：35 days

## Task Status

**Claimant's Github ID**：yinfan98

**Claim Time**：2023年10月25日

**PR Link**：None

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by FlamingoPg** [comment]
Claim the mission

**Comment by BaophanN** [comment]
Sorry, where is the docker prebuilt image released?
